.travis.yml

```diff
@@ -58,7 +58,7 @@ jobs:
         - travis_retry composer require --dev --no-update phpstan/phpstan:^0.9
       install: travis_retry composer update --prefer-dist --prefer-stable
       script:
-        - vendor/bin/phpstan analyse --level=0 -c phpstan.neon src
+        - vendor/bin/phpstan analyse --level=1 -c phpstan.neon src
         - vendor/bin/phpstan analyse --level=1 -c phpstan-tests.neon tests
```

Comment:
Why not combine them?
```sh
vendor/bin/phpstan analyse --level=1 -c phpstan.neon src tests
```

Reply:
This allows to:
1) Upgrade level of each (src / tests) separately, see https://github.com/sebastianbergmann/phpunit/blob/114a7630770a5e68e6959a6ea6a53dd0094b2999/.travis.yml#L61-L62
2) make it clear which things are ignored because of src and which are ignored because of tests